sort_names.py

```diff
@@ -1,95 +1,188 @@
         csvreader = csv.reader(
             file, delimiter=","
         )  # delimiter="," - if the data is concatenated in the first column by ','
         # delimiter=";" - if the data in the different column
+        first_row = next(csvreader)
         read_file = list(csvreader)
-        num_people = len(read_file) - 1
+        num_people = len(read_file)
         print("--- START ---")
-        print("Estimated time ~ {} seconds".format(num_people * 1.2))
+        print(f"Estimated time ~ {round(num_people * 1.2 / 60)} minutes")
+        writer.writerow(first_row)
+        writer2.writerow(first_row)
 
         for row in read_file:
-            if (
-                (row[24] not in first_names)
-                and (row[25] not in last_names)
-                and (forbidden_word not in row[43])
-                and face_filter(row[30], limit_age, races)
-            ):
+            result, t_counters = conditions(
+                row, d_rases, first_names, last_names, positions
+            )
+            counters(d_counters, t_counters)
+            if result:
                 writer.writerow(row)
-                count += 1
+            else:
+                writer2.writerow(row)
             count_row += 1
             condition = round(100 * count_row / num_people)
 
-            if (condition % 5 == 0) and (n_cond != condition):
+            if (condition % 10 == 0) and (n_cond != condition):
                 n_cond = condition
-                print("---- {}% completed ----".format(n_cond))
+                print(f"---- {n_cond}% completed ----")
         print("There were", num_people, "people before filtering")
-        print("Done, now there are", count, "people")
-        print("Results at the file '{}'".format(res_file_name))
-
-
-def input_black_list():
-    """Read .csv black list and return list of first_names and list of last_names"""
-    with open(black_list, "r", encoding="utf8") as file:
-        csvreader = csv.reader(file, delimiter=",")
-        first_names = []
-        last_names = []
-        title_t = next(csvreader)
-        for row in csvreader:
-            first_names.append(row[0])
-            last_names.append(row[1])
-    return first_names, last_names
-
-
-def face_filter(img, age=25, race="white"):
-    """Filter people by age and race using photo
-
-    img - path to photo
-    age - no younger than "age"
-    race - only one race from ['asian', 'indian', 'black', 'white', 'middle eastern', 'latino hispanic']
-    """
-    try:
-        obj = DeepFace.analyze(img_path=img, actions=["age", "race"])
-        return (obj["age"] > age) and (obj["dominant_race"] in races)
-    except Exception as _ex:
-        return _ex
+        print("Done, now there are", d_counters["count"], "people")
+        print(
+            f"Results at the file '{res_file_name}', deleted people at the file '{res_del_file_name}'"
+        )
+        print("Statistic:")
+        if DEL_PEOPLE_WITHOUT_AVATAR:
+            print(
```

Comment:
please form an output string first, and use only one print statement in the end.

Reply:
done, Thank you! I use the list to combine all information